lib/web/apiserver_test.go

```diff
@@ -964,6 +964,40 @@ func (s *WebSuite) TestTerminal(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (s *WebSuite) TestWebsocketPingLoop(c *C) {
+	// change cluster default config for keep alive interval to be ran faster
+	clusterConfig, err := services.NewClusterConfig(services.ClusterConfigSpecV3{
+		SessionRecording:    services.RecordAtNode,
+		ProxyChecksHostKeys: services.HostKeyCheckYes,
+		KeepAliveInterval:   services.NewDuration(250 * time.Millisecond),
```

Comment:
Any reason this can't just be `1ms`?  AFAIK we don't actually err out on late/missing `pong` frames.

Reply:
I feel like sending 1000 keepalives a second versus 4 might be a bit excessive...